tests/executables_src/testLMapBackendUnified.cpp

```diff
@@ -323,12 +332,17 @@ struct RegSingleWordB : ScalarRegisterDescriptorBase<RegSingleWordB> {
 struct RegSingleWord_push : ScalarRegisterDescriptorBase<RegSingleWord_push> {
   std::string path() { return "/SingleWord_push"; }
   bool isPush() { return true; }
+  bool isWriteable() {
+    std::cout << "Warning: Writing test for /SingleWord_push has been disabled due to missing support in the dummy."
+              << std::endl;
+    return false;
+  }
 
   const uint32_t increment = 3;
 
   typedef uint32_t minimumUserType;
   typedef int32_t rawUserType;
-  DummyRegisterAccessor<minimumUserType> acc{exceptionDummy.get(), "", "/BOARD.WORD_FIRMWARE"};
+  DummyRegisterAccessor<minimumUserType> acc{exceptionDummy3.get(), "", "/BOARD.WORD_FIRMWARE"};
```

Comment:
In a future change, can we somehow properly name the exception dummies? Numbering them is a bit confusing further down on what the difference between 1,2 and 3 is

Reply:
Probably yes. I will leave a comment at the creation point.
Although dummy3 is a
- I just needed the same register in another backend
- I hacked the push-type into this map file because dummy 1 is the re-used mtcadummy, and the mtcadummy does not support push type

Reply:
I renamed it to make it a bit clearer. In the lmap it's still called target1, 2, 3, though.